governance/paper-process.md

```diff
@@ -0,0 +1,169 @@
+content:
+* Who do we expect to read the paper?
+* What positions do they hold?
+* Why are we writing this paper?
+* What problem are we intending to resolve?
+* What assumptions are we making about the audience or the expected content of the paper?
+    * Note - these are documented then in the "Introduction > Assumptions" section of the paper
+* What are our Goals and Non-Goals?
+    * Note - these are documented then in the "Introduction > Scope > Goals" and "Introduction > Scope > Non-Goals"
+* Is the scope in the issue still representative of what we intend to cover?
+
+### Tasking assignment
+
+Contributors should highlight and assign themselves to various areas of the
+outline.  This allows contributors to focus on a few areas and identify others
+where more contributors is needed.  If using a google doc for the paper,
+assignments may be done through the "assign to me" feature after highlight the
+area or by using the "+foo@bar.com" comment feature for the assignee's email.
+Once task, content can begin rough-in
+
+### Content rough-in
+
+Assignees can provide content rough-in in a variety of ways.  They may provide
+raw content in the former of phrases, disjointed paragraphs, bullets on the
+topic, or they draft content in the form of completed paragraphs.  If leveraging
+draft content, placing this content in quotes helps other reviewers know it is
+'near-complete' and can review it as a wholistic section.
+
+### Collaborative review
+
+Contributors now review all content roughed-in and transition raw content into
+draft content.  They may begin commenting on sections, expanding ideas, refining
+arguments, and providing citations.  Contributors are encouraged to discuss
+content specific items in the doc through comments whereas larger themes and
+writing decisions should be discussed in the corresponding slack channel.  This
+keeps the doc clean as well as documents thematic decisions in the primary
+collaboration & coordination channel.
+
+### Executive summary and content wrap-up
+
+If the paper exceeds five (5) pages, an Executive summary commensurate with the
+length of the document (but not to exceed 3 pages) should be written.  Content,
+comments, and other discussions should be closed out and finalized at this time.
+
+### Narrative voice
+
+No more than three contributing individuals should be assigned for narrative
+voice.  The narrative voice is a semi-final pass of the paper to ensure it reads
+as a single, unified voice.  It should ensure:
+* the language origin is consistent throughout the document (lang_en or lang_us),
+* phrasing is similar (caddy corner not mixed in with kitty corner),
+* acronyms are spelled out at their first use and then abbrieviated later,
+* footnotes and citations are consistent and not direct hyperlinks in the text
+* vague terms are defined in a gloassary or otherwise cited to the cloud native security lexicon in the repo
```

Comment:
```suggestion
* vague terms are defined in a glossary or otherwise cited to the cloud-native security lexicon in the repo
```

Reply:
```suggestion
* vague terms are defined in a glossary or otherwise cited to the cloud native security lexicon in the repo
```

Reply:
@thschue thanks for the comments!! I removed the hyphen from cloud native as the TOC repo definition does not hyphenate the term.